pkg/gameservers/portallocator.go

```diff
@@ -161,6 +163,27 @@ func (pa *PortAllocator) Allocate(gs *agonesv1.GameServer) *agonesv1.GameServer
 				if p.PortPolicy == agonesv1.Passthrough {
 					gs.Spec.Ports[i].ContainerPort = a.port
 				}
+
+				// create a second port for UDP
```

Comment:
This comment says "udp" but the code inside the block creates a port with "-tcp" appended, which I find a bit confusing. Can you clarify this comment?